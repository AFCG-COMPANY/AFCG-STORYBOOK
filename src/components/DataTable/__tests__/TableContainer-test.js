import React from 'react';
import { mount } from 'enzyme';
import { TableContainer } from '../';

describe('DataTable.TableContainer', () => {
  it('should render', () => {
    const wrapper = mount(<TableContainer className="custom-class" />);
    expect(wrapper).toMatchSnapshot();
  });
});
