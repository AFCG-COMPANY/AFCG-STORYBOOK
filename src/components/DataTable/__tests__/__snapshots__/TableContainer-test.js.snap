// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTable.TableContainer should render 1`] = `
<TableContainer
  className="custom-class"
>
  <div
    className="custom-class bx--data-table-v2-container"
  />
</TableContainer>
`;
